### ISSUE TITLE
Comments on Messages should display most recent on top

### ISSUE THREAD

**Issue by Whiteaj36** [VentApp/vent-rails#9]
While counter-intuitive at first. A fast streaming view of comments/live reacts to a message that you can see without scrolling, I think fits the spirit of the app.

**Comment by robwagner33** [comment]
Yeah... I think we will want to talk about this. I like the idea of a live stream, but I also think bubbling up popular comments is a good idea. We should discuss.
